lms/djangoapps/courseware/tests/test_video_mongo.py

```diff
@@ -866,6 +871,80 @@ def test_source_is_empty(self):
         self.assertFalse(self.item_descriptor.download_video)
 
 
+@attr('shard_1')
+class TestEditorSavedMethod(BaseTestXmodule):
+    """
+    Make sure that `editor_saved` method works correctly.
+    """
+    CATEGORY = "video"
+    DATA = SOURCE_XML
+    METADATA = {}
+
+    def setUp(self):
+        super(TestEditorSavedMethod, self).setUp()
+        self.setup_course()
+
+    def test_editor_saved_when_html5_sub_not_exist(self):
+        """
+        When there is youtube_sub exist but no html5_sub present for
+        html5_sources, editor_saved function will generate new html5_sub
+        for video.
+        """
+        metadata = {
+            'source': 'http://youtu.be/3_yD_cEKoCk',
+            'html5_sources': ['http://example.org/video.mp4'],
+        }
+        # path to subs_3_yD_cEKoCk.srt.sjson file
+        file_name = 'subs_3_yD_cEKoCk.srt.sjson'
+        # pylint: disable=no-value-for-parameter
+        test_dir = path(__file__).abspath().dirname().dirname().dirname().dirname().dirname()
```

Comment:
Just going to specific test folder using forward moving ```dirname()```, this is also written in other test cases but can change the test_dir to fixed string.